doc/References.rst

```diff
@@ -0,0 +1,4 @@
+References
+##########
+
+* :ghrepo:`olofk/ipyxact`
```

Comment:
What about these ?
* https://github.com/edaa-org/IPXACT-Schema
* https://pypi.org/project/pyIPXACT/ (renamed to this repository)
* https://pypi.org/project/peakrdl-ipxact/
* https://pypi.org/project/ralbot-ipxact/
* https://pypi.org/project/ipxact2systemverilog/
* https://github.com/iDoka/ipxact-cli-tools

Do we need to reference Kactus2?

Reply:
No judgement from my side, I just found them and listed them here in the comment.

Reply:
I will add them as well.

We do not *need to* reference Kactus2, but I think we should do it :D I will ad it too.

Reply:
Done.